hooks/build

```diff
@@ -1,10 +1,53 @@
 #!/bin/bash
+set -eo pipefail
 
-# Enable BuildKit
+# Enable Docker BuildKit
 export DOCKER_BUILDKIT=1
 
-# Extract the Git tag from the SOURCE_COMMIT
+# Platforms to build for
+PLATFORMS="linux/amd64,linux/arm64"
+
+# Check if SOURCE_COMMIT is set
+if [ -z "${SOURCE_COMMIT}" ]; then
+  echo "SOURCE_COMMIT not set. Unable to determine version."
+fi
```

Comment:
If `SOURCE_COMMIT` is required for versioning, consider exiting early instead of just printing a warning:
```bash
if [ -z "${SOURCE_COMMIT}" ]; then
  echo "SOURCE_COMMIT not set. Aborting."
  exit 1
fi
```
Or fallback to something like `v0.0.0-dev` if local builds without it are expected.